report-jtreg-comparator/src/test/java/io/jenkins/plugins/report/jtreg/main/comparator/StackTraceCompareTest.java

```diff
@@ -1,13 +1,73 @@
 package io.jenkins.plugins.report.jtreg.main.comparator;
 
+import io.jenkins.plugins.report.jtreg.main.comparator.arguments.ComparatorArgDeclaration;
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
-import static org.junit.jupiter.api.Assertions.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 
 class StackTraceCompareTest {
 
+    @Test()
+    void removeSetReferentialNull() {
+        //Assertions.assertEquals(, result);
+        Exception thrown = Assertions.assertThrows(
+                Exception.class,
+                () -> StackTraceCompare.removeReferentialFromArgs(null),
+                "Expected Exception with null input"
+        );
+    }
+
     @Test
-    void removeSetReferential() {
-        
+    void setReferential() {
+
+        List<String> result1 = StackTraceCompare.removeReferentialFromArgs(new ArrayList<>());
+        Assertions.assertEquals(new ArrayList<>(), result1);
+
+        List<String> result2 = StackTraceCompare.removeReferentialFromArgs(Arrays.asList("Buenos Aires", "Córdoba", "La Plata"));
+        Assertions.assertEquals(Arrays.asList("Buenos Aires", "Córdoba", "La Plata"), result2);
+
```

Comment:
it would be much better to use instead of south america cities, which come form Stack Owerflow , soem more real switches , isnt it?-)